karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
+// Generated on Fri Jan 22 2016 15:42:55 GMT+0000 (GMT Standard Time)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
```

Comment:
This works just fine as it is, but IMO, it would be be more consistent if it used 'mocha' and 'browserify' since the others all work that way. Feel free to ignore though :smile: